src/hooks/featured-place-view-manager-hooks.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+
+export function useFeaturedPlaceViewCameraChange(view, selectedFeaturedPlace, featuredPlacesLayer) {
+const [coords, setCoords] = useState(null);
+useEffect(() => {
+  if (selectedFeaturedPlace && featuredPlacesLayer) {
+    const query = featuredPlacesLayer.createQuery();
+    query.where = `nam_slg = '${selectedFeaturedPlace}'`
+    featuredPlacesLayer.queryFeatures(query).then(result => {
+      const { lon, lat } = result.features[0].attributes;
```

Comment:
Not sure if `get` is safer in this situation, if some property on the path is undefined an error will trigger when trying to destructure. I would maybe avoid importing `lodash` in this case. What do you think @aabdaab ?
 From `lodash` REPL:
![image](https://user-images.githubusercontent.com/6906348/63346924-53ad9780-c356-11e9-824c-9fcb878fe949.png)

Reply:
in the case, you showed, yes, destructuring would throw an error, same as here:
```
const { lon, lat } = result.features[0].attributes;
```
To take advantage of `get` we would have to change it to:
```
const lon = _.get(result, 'features[0].attributes.lon');
const lat = _.get(result, 'features[0].attributes.lat');
```

![image](https://user-images.githubusercontent.com/6136899/63351968-2235cb80-c358-11e9-95f8-07672fdd2c64.png)

I don't have a strong opinion about this although I'm a bit worried not checking for undefined in deeply nested objects (i think it's better to set an undefined just in case than throw an error to a user)

Reply:
Agree 👍